flake.nix

```diff
@@ -1,11 +1,11 @@
 {
   description = "The purely functional package manager";
 
-  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/release-24.11";
```

Comment:
Out of curiosity, why is this using the master-branch-equivalent of 24.11 instead of, say, `nixpkgs-24.11` or `nixos-24.11`?